CONTRIBUTING.md

```diff
@@ -0,0 +1,188 @@
+# Contributing
+
+This document provides guidelines for contributing to samples.
+
+## Dependencies
+
+The following dependencies must be installed on the development system:
+
+- [Docker Engine][docker-engine]
+- [Google Cloud SDK][google-cloud-sdk]
+- [make]
+
```

Comment:
Can we mention that Cloud Shell already meets these requirements, so we recommend using Cloud Shell?

Reply:
done